src/docs/node-integration.md

```diff
@@ -40,11 +40,13 @@ Fork [V8’s Node.js repository on GitHub](https://github.com/v8/node/) (click t
 Add your both your fork and V8’s fork as remotes to the existing checkout:
 
 ```bash
+cd node
 git remote add v8 http://github.com/v8/node
 git remote add <your-user-name> http://github.com/<your-user-name>/node
 git fetch
 git checkout node-ci-<sync-date>
-git checkout -b make-changes
+export BRANCH_NAME=`date +"%Y-%m-%d"`_fix_name
```

Comment:
Is this something you intended to include in this PR? The description doesn't mention it.

Reply:
yes I added this on purpose to make it cleaner when merging branches in.